modules/osdk-create-cr.adoc

```diff
@@ -200,6 +200,13 @@ memcached-sample                        5/5     5            5           3m
 ----
 endif::[]
 
+. Delete the CR by running the following command:
```

Comment:
Better to change to "Delete the CR:", no need to add "by running the following command"

Reply:
@Xia-Zhao-rh As per the OS docs guidelines, it is better to mention "by running the following command" for clarity.

Reply:
OK, I hava no other comments. lgtm